code/_core/obj/item/container/food/fish.dm

```diff
@@ -0,0 +1,115 @@
+/obj/item/container/food/fishing
+	name = "fish"
+	desc = "morning!! if it was a GOOD morning i'd be FISHING"
+	desc_extended = "A fresh catch."
+	icon = 'icons/obj/item/consumable/food/fish.dmi'
+	icon_state = "fish"
+	health = /health/obj/item/misc/
+	scale_sprite = FALSE
+
+/obj/item/container/food/fishing/can_be_attacked(var/atom/attacker,var/atom/weapon,var/params,var/damagetype/damage_type)
+	return TRUE
+
+/obj/item/container/food/fishing/on_damage_received(var/atom/atom_damaged,var/atom/attacker,var/atom/weapon,var/list/damage_table,var/damage_amount,var/critical_hit_multiplier,var/stealthy=FALSE)

```

Comment:
I think this would be better that it changed its icon state instead or something instead of creating a new type, but this is something I could do myself.

Reply:
for the fish, originally i made it seperate in case i wanted to make some special which i did for the goldfish or required for crafting _but mainly cuz that was the only way my baby brain could put it in really PLUS its really funny to beat up fish for no raeson_ but yeah if you could go ahead, thanks